cobbler/modules/managers/import_signatures.py

```diff
@@ -1015,6 +924,131 @@ def rsync_repo_adder(distribution: "Distro") -> None:
         """
         return
 
+    # ==========================================================================
+    # windows-specific
+
+    def import_winpe(self) -> None:
```

Comment:
Codacy says that this method is:

- quite large,
- has too many local variables,
- has too many statements and
- is too complex

If it is reasonable I would kindly request that we split this into multiple methods.

Reply:
I split this large method into a set of smaller ones and changed the logic a little.